gulp-tasks/utils/upload-bundle-to-cdn.js

```diff
@@ -0,0 +1,34 @@
+const path = require('path');
+const glob = require('glob');
+
+const spawnPromiseWrapper = require('./spawn-promise-wrapper');
+
+const PROJECT_ID = 'workbox-bab1f';
+const BUCKET_NAME = 'workbox-cdn';
+
+module.exports = (tagName, directoryToUpload) => {
+  return spawnPromiseWrapper('gcloud', [
+    'config', 'set', 'project', PROJECT_ID,
+  ])
+  .then(() => {
+    const globPattern = path.posix.join(
+      directoryToUpload, '*');
+    const filePaths = glob.sync(globPattern, {
+      absolute: true,
+    });
+
+    return filePaths.reduce((promiseChain, filePath) => {
```

Comment:
Prefer `for ... of` + `await`: https://stackoverflow.com/a/37576787/385997 (or switch to the `node` client library).